podcasts/DiscoverViewController+CategoryRedesign.swift

```diff
@@ -0,0 +1,70 @@
+import PocketCastsServer
+
+private let popularItemsCount = 5
```

Comment:
Instead of this why no declare an enum inside the category like this:

```
extension DiscoverViewController {
    private enum Constant {
        static let popularItemsCount = 5
    }
...
```